clients/integration-tests/src/test/java/org/apache/kafka/clients/admin/AdminClientRebootstrapTest.java

```diff
@@ -33,6 +32,9 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Stream;
 
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+
 public class AdminClientRebootstrapTest {
```

Comment:
@mumrah could you please consider using a different test? `AdminClientRebootstrapTest` requires some rewriting (see https://github.com/apache/kafka/pull/19094#discussion_r1986208718). Perhaps we could use `AdminFenceProducersTest` instead?